adapter/src/main/java/org/jvalue/ods/adapterservice/datasource/event/DatasourceUpdateEvent.java

```diff
@@ -0,0 +1,41 @@
+package org.jvalue.ods.adapterservice.datasource.event;
+
+import org.jvalue.ods.adapterservice.datasource.model.Datasource;
+import org.jvalue.ods.adapterservice.datasource.model.DatasourceTrigger;
+
+import java.util.Objects;
+
+public class DatasourceUpdateEvent extends DatasourceConfigEvent {
```

Comment:
Exact duplicate of CreateEvent? => then combine to one

Reply:
Right, this is redundant. I will simplify the config events by using only one for creation, update and deletion.